src/main/java/com/mraof/minestuck/player/ClientPlayerData.java

```diff
@@ -28,7 +28,8 @@
 	
 	private static Modus modus;
 	private static Title title;
-	private static int rung;
+	
+	public static int rung;
```

Comment:
is there a reason for making this public, and for adding an extra line?

Reply:
yes

Reply:
yes